src/device-registry/routes/api-v1.js

```diff
@@ -381,6 +381,16 @@ router.post(
         .withMessage(
           "the mountType value is not among the expected ones which include: pole, wall, faceboard, suspended and rooftop "
         ),
+      body("category")
+        .exists()
```

Comment:
Category is now a required field during device creation! This is to ensure that we are able to utilise different config mappings based on device category.